### DISCUSSION TITLE
makeRequest: don't send parameters with null value?

### DISCUSSION THREAD

**Discussion by zraly** [jiripudil/Naja#253] [Support] [answered]
Regarding `naja.makeRequest(method, url, data = null, options = {})`:

If data has this value:

```javascript
data = {
  name: 'David',
  surname: null
}
```

`surname` is still sent as a string `null`. 

But I would like NOT to send items with null values. Would make this any sense or it's not a good idea?

Thanks

**Comment by jiripudil** [comment, marked as answer]
Yep, makes complete sense, I'd say the current behaviour is a bug. I've created an issue for it (#254). Thanks for the report!

**Reply by zraly** [reply]
Great! Thanks.
